### PR TITLE
enable sqlite vm instructions config in docker

### PR DESCRIPTION
**Pull request by prettyirrelevant**

### CODE REVIEW

File: packaging/docker/entrypoint.py [hunk 1]
```diff
@@ -71,13 +71,15 @@ def load_config_from_env() -> Dict[str, Any]:
     sleep_secs = os.environ.get('SLEEP_SECS')
     max_size_in_mb_all_logs = os.environ.get('MAX_SIZE_IN_MB_ALL_LOGS')
     max_logfiles_num = os.environ.get('MAX_LOGFILES_NUM')
+    sqlite_instructions = os.environ.get('SQLITE_VM_INSTRUCTIONS')
```
Comment (LefterisJP) [inline packaging/docker/entrypoint.py:74]: Why not name it the env var the same?

Comment (prettyirrelevant) [reply packaging/docker/entrypoint.py:74]: I'll just stick to `SQLITE_INSTRUCTIONS` then.

Comment (kelsos) [reply packaging/docker/entrypoint.py:74]: Yes it would be better to keep the same name

**Review by kelsos — APPROVED** [review]
lgtm

**Review by LefterisJP — APPROVED** [review]

**Pull request merged**